server/routes/api/choropleth.py

```diff
@@ -159,7 +160,11 @@ def choropleth_geo():
     # Return as json payload.
     return jsonify({
         "type": "FeatureCollection",
-        "features": features
+        "features": features,
+        "properties": {
+            # TODO(iancostello): Don't just pick the first and check.
+            "current_geo": names_by_geo[requested_geoDcid][0]
```

Comment:
There might not be a result for the requested dcid - I suppose that's what the TODO is? It wasn't clear.

Reply:
Yes that was the TODO. I am going to address all these TODOs in the next PR